ListingManager/ListingManager.cs

```diff
@@ -24,15 +40,15 @@ public static IEnumerable<string> GetAllExtraListings(string pathToStartFrom)
         }
     }
 
-    private static bool TryGetListing(string listingPath, out ListingInformation? listingData)
+    private static bool TryGetListing(string listingPath, out ListingInformation? listingData, bool isTest = false)
```

Comment:
It is a bit awkward to have an out parameter listed before an in parameter. I would suggest, just doing a method overload instead.

Reply:
I agree, fixed